ngraph/python/tests/test_ngraph/test_create_op.py

```diff
@@ -77,6 +77,46 @@ def test_ctc_greedy_decoder(dtype):
     assert list(node.get_output_shape(0)) == expected_shape
 
 
+@pytest.mark.parametrize("fp_dtype, int_dtype, int_ci, int_sl, merge_repeated, blank_index",
+                         [
+                             (np.float32, np.int32, "i32", "i32", True, True),
+                             (np.float32, np.int32, "i64", "i32", True, True),
+                             (np.float32, np.int32, "i32", "i64", True, True),
+                             (np.float32, np.int32, "i64", "i64", True, True),
+                             (np.float64, np.int64, "i32", "i32", False, True),
+                             (np.float64, np.int64, "i64", "i32", False, True),
+                             (np.float64, np.int64, "i32", "i64", False, True),
+                             (np.float64, np.int64, "i64", "i64", False, True),
+                             (np.float32, np.int32, "i32", "i32", True, False),
+                             (np.float32, np.int32, "i64", "i32", True, False),
+                             (np.float32, np.int32, "i32", "i64", True, False),
+                             (np.float32, np.int32, "i64", "i64", True, False),
+                             (np.float64, np.int64, "i32", "i32", False, False),
+                             (np.float64, np.int64, "i64", "i32", False, False),
+                             (np.float64, np.int64, "i32", "i64", False, False),
+                             (np.float64, np.int64, "i64", "i64", False, False)
+                         ],)
+def test_ctc_greedy_decoder_seq_len(fp_dtype, int_dtype, int_ci, int_sl, merge_repeated, blank_index):
+    input0_shape = [8, 20, 128]
+    input1_shape = [8]
+    input2_shape = [1]
+    expected_shape = [8, 20]
+
+    parameter_input0 = ng.parameter(input0_shape, name="Input0", dtype=fp_dtype)
+    parameter_input1 = ng.parameter(input1_shape, name="Input1", dtype=int_dtype)
+    parameter_input2 = ng.parameter(input2_shape, name="Input2", dtype=int_dtype)
+    if blank_index is False:
+        parameter_input2 = input0_shape[2] - 1
```

Comment:
You get the idea right. However this should be implemented in _ngraph/python/src/ngraph/opset6/ops.py_ inside _ctc_greedy_decoder_seq_len_ function. Right now all we have is a test mock that does not work across nGraph module thus outside of this this test `None` value will still throw errors.